main/commons/src/main/java/org/cryptomator/common/vaults/WebDavVolume.java

```diff
@@ -73,12 +73,22 @@ private void mount() throws VolumeException {
 	}
 
 	@Override
-	public void reveal(Revealer r) throws VolumeException {
+	public void reveal(RevealerFacade r) throws VolumeException {
 		try {
-			mount.reveal(r);
+			mount.reveal(p -> {
+				try {
+					r.reveal(p);
+				} catch (VolumeException e) {
+					throw new Mounter.RevealException(e);
+				}
+			});
```

Comment:
Ah crap... Didn't think of this problem when I decided we should define a checked `RevealException` in each of the three *-nio-adapters.

I think we should discuss this in a call.

Reply:
Resolved in 81fb6b8